src/core/bpf/restrict_ifaces/restrict-ifaces.bpf.c

```diff
@@ -6,7 +6,7 @@
 #include <linux/bpf.h>
 #include <bpf/bpf_helpers.h>
 
-const volatile __u8 is_allow_list = 0;
+const __u8 is_allow_list SEC(".rodata") = 0;
```

Comment:
The `SEC` macro also sets the [`used`](https://clang.llvm.org/docs/AttributeReference.html#used) attribute, is my understanding correct that this accomplishes the same purpose as `volatile` in regards to preventing the variable from being optimized away?

Reply:
no, used attribute just prevents from compiler complaining that there is no explicit usage for given variable. We rely on used attribute more for BPF program declarations (and something for map declarations), less so for global variables. But we utilize the same SEC() macro for all of them.